apps/web/src/components/file-browser/file-browser-data.tsx

```
@@ -6,28 +6,69 @@ import {
+		return sortConfig.direction === "asc" ? 
+			<ArrowUp className="h-3 w-3" /> : 
+			<ArrowDown className="h-3 w-3" />;
+	};
+
+	const SortableHeader = ({ field, children }: { field: SortField; children: React.ReactNode }) => (
+		<th className="p-3">
+			<button
+				onClick={() => handleSort(field)}
+				className="flex items-center gap-1 hover:text-foreground transition-colors cursor-pointer"
+				disabled={!onSortChange}
+			>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)